l2-contracts/contracts/bridge/L2SharedBridge.sol

```diff
@@ -45,7 +48,7 @@ contract L2SharedBridge is IL2SharedBridge, IL2SharedBridgeLegacy, Initializable
     IL2StandardDeployer public standardDeployer;
```

Comment:
Maybe let's spend some time on another name.
As it's a vault as well, which can be reused on other hyperchains.

L2StandardTokenVault?